### PR TITLE
Allow the AI to turn on red alert (without ctrl click now)

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
This PR allows the AI to turn on swipe related things such as red alert using ctrl click.

## Why It's Good For The Game
This PR raison d'être is that everyone keeps freaking dying, leaving the CIC deserted at the times we need it the most. 
Being that the AI is pretty much 70% of shipside command at any given game, it seems silly for them not to be able to u

## Changelog
:cl:
expansion: Allows the AI to use keycard devices using ctrl click
/:cl:

### CODE REVIEW

**Comment by BraveMole** [comment]
Alt click will probably be very obscure, why not just clicking on the keycart auth? That could show a pop up asking you if you really want to go red alert?

**Comment by Ieteihowmee** [comment]
Please, this is absolutely needed.

**Comment by lbnesquik** [comment]
> 
> 
> Alt click will probably be very obscure, why not just clicking on the keycart auth? That could show a pop up asking you if you really want to go red alert?

I could try that, but i'd need to make an interface. I think.

**Comment by lbnesquik** [comment]
I made the interface.

File: code/modules/security_levels/keycard_authentication.dm [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	var/obj/machinery/keycard_auth/event_source
 	var/mob/event_triggered_by
 	var/mob/event_confirmed_by
+	var/synth_activation = 0
```
Comment (SplinterGP) [inline code/modules/security_levels/keycard_authentication.dm:15]: autodoc

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by SplinterGP — APPROVED** [review]

**Comment by Ieteihowmee** [comment]
Good work!

**Pull request merged**